include/staff/department.inc.php

```diff
@@ -312,6 +312,7 @@
                 <select name="group_membership">
 <?php foreach (array(
     Dept::ALERTS_DISABLED =>        __("No one (disable Alerts and Notices)"),
+    Dept::ALERTS_ADMIN_ONLY =>       __("Admin Only"),
```

Comment:
I'll change wording here to `Admin Email Only` - to avoid, "but I'm Admin" type of complaints.

Reply:
Updated!